README.md

```diff
@@ -62,17 +62,18 @@ You can also explore provided examples in the [example](./example/) directory to
 
 # Supported Libraries
 
-| Plugin Name  | Repository Url                             | Min Supported Version | Max Supported Version |
-|--------------|--------------------------------------------|-----------------------|-----------------------|
-| database/sql | https://pkg.go.dev/database/sql            | -                     | -                     |
-| gin          | https://github.com/gin-gonic/gin           | v1.7.0                | v1.10.0               |
-| go-redis     | https://github.com/redis/go-redis          | v9.0.5                | v9.5.1                |
-| gorm         | https://github.com/go-gorm/gorm            | v1.22.0               | v1.25.9               |
-| logrus       | https://github.com/sirupsen/logrus         | v1.5.0                | v1.9.3                |
-| mongodb      | https://github.com/mongodb/mongo-go-driver | v1.11.1               | v1.15.2               |
-| mux          | https://github.com/go-gorm/gorm            | v1.3.0                | v1.8.1                |
-| net/http     | https://pkg.go.dev/net/http                | -                     | -                     |
-| zap          | https://github.com/uber-go/zap             | v1.20.0               | v1.27.0               |
+| Plugin Name  | Repository Url                              | Min Supported Version | Max Supported Version |
+|--------------|---------------------------------------------|-----------------------|-----------------------|
+| database/sql | https://pkg.go.dev/database/sql             | -                     | -                     |
+| gin          | https://github.com/gin-gonic/gin            | v1.7.0                | v1.10.0               |
+| go-redis     | https://github.com/redis/go-redis           | v9.0.5                | v9.5.1                |
+| gorm         | https://github.com/go-gorm/gorm             | v1.22.0               | v1.25.9               |
+| logrus       | https://github.com/sirupsen/logrus          | v1.5.0                | v1.9.3                |
+| mongodb      | https://github.com/mongodb/mongo-go-driver  | v1.11.1               | v1.15.2               |
+| mux          | https://github.com/gorilla/mux              | v1.3.0                | v1.8.1                |
+| echo         | https://github.com/labstack/echo            | v4.0.0                | v4.12.0               |
```

Comment:
You may add the echo after `database/sql` to make the whole table conform to dictionary order